pkg/otp/code.go

```diff
@@ -45,40 +52,68 @@ func GenerateCode(length uint) (string, error) {
 	return result, nil
 }
 
+// GenerateAlphanumericCode will generate an alpha numberic code.
```

Comment:
We should _probably_ use sethvargo/go-password here instead. That was reviewed by ISE and bradfitz.

Reply:
that didn't quite have what I wanted - I did make this better though using that as a basis.